### ISSUE TITLE
Incorrectly dealing with batch normalization layers

### ISSUE THREAD

**Issue by ainrichman** [maxpumperla/elephas#57]
The elephas calculates the deltas of the weights by simple subtraction. This strategy is efficient for the conv2d layers and dense layers. However, modern neural networks always contain batch normalization layers in which the subtraction strategy fails.  Thus, elephas could not be used for distributed training some popular networks such as google inceptions and YOLO. 

Can this be solved?

**Comment by danielenricocahall** [comment]
Hello @ainrichman, would you happen to have suggestions on how to correctly implement distributed BN?

**Comment by danielenricocahall** [comment]
Copied to new fork https://github.com/danielenricocahall/elephas/issues/18, closing here!
